dist/css/pulsar.css

```diff
@@ -3173,7 +3173,7 @@ body {
   text-align: center;
```

Comment:
Why are the /dist files here, thought we had that in .gitignore... https://github.com/jadu/pulsar/blob/develop/.gitignore#L17

Reply:
You're linking pulsar there, we've always committed compiled styles to the docs.